pkg/cloudprovider/aws/aws.go

```diff
@@ -22,18 +22,26 @@ import (
 	"net"
 	"regexp"
 	"strings"
+	"sync"
 
 	"code.google.com/p/gcfg"
 	"github.com/mitchellh/goamz/aws"
 	"github.com/mitchellh/goamz/ec2"
+	"github.com/mitchellh/goamz/elb"
```

Comment:
Yes, it is a bit tricky to get the ELB dependency to land.  I tried to commit it as a different PR and the problem is that Godeps sees it is unused and so removes it.  I think realistically it has to be done in this PR, although I will split it out into a separate commit.